runtime/lib/app-runtime.js

```diff
@@ -325,9 +321,9 @@ AppRuntime.prototype.resetNetwork = function resetNetwork (options) {
     this.component.lifetime.destroyAll(),
     this.setMicMute(false, { silent: true })
   ]).then(() => this.component.custodian.resetNetwork(options))
-    .then(deferred, err => {
+    .catch(err => {
```

Comment:
Why `deferred` was removed on thenable case?

Reply:
The light will be replaced by the setupNetwork effect because `shouldResume` is false, if we stop it here,  the system standby effect will be displayed, that is not right.

Reply:
@FeelyChau Could you please change the flow only? Just keep the deferred still?

Reply:
Update.

Reply:
Clear now, so your change says only if the network failed, the long pressed effect needs to be reset. There are some cases the standby light are not playing, so this effect could not be released.

BTW, @FeelyChau what problem do you wanna resolve in this PR? I don't see any problems that showing standby effect finally after reset network.

Reply:
> There are some cases the standby light are not playing, so this effect could not be released.

Or we should set timeout for stopping the light?